projects/uitdatabank/docs/search-api/embedding.md

```diff
@@ -0,0 +1,292 @@
+      {
+         "@id":"https://io.uitdatabank.be/event/d9a71b53-1756-4126-9926-a83f5dd84f45",
+         "@type":"Event"
+      },
+      {
+         "@id":"https://io.uitdatabank.be/place/557d0ddc-efc9-42b3-934b-9f88b0945ab1",
+         "@type":"Place"
+      },
+      "..."
+   ]
+}
+```
+
+#### With embedding
+**request**
+```
+GET /offers/?embed=true
+```
+
+**response**
+
+The response contains actual JSON-LD documents.
+```js
+{
+  "@context": "http://www.w3.org/ns/hydra/context.jsonld",
+  "@type": "PagedCollection",
+  "itemsPerPage": 30,
+  "totalItems": 12,
+  "member": [
+    {
+      "@id": "https://io.uitdatabank.be/place/39e6d5ee-c3d6-453a-bcb5-4e6e0eaf7054",
+      "@type": "Place",
+      "@context": "/contexts/place",
+      "name": {
+        "nl": "Jeugdhuis Sojo"
+      },
+      "address": {
+        "addressCountry": "BE",
+        "addressLocality": "Kessel-Lo (Leuven)",
+        "postalCode": "3010",
+        "streetAddress": "Eenmeilaan 1"
+      },
+      "terms": [
+        {
+          "id": "JCjA0i5COUmdjMwcyjNAFA",
+          "label": "Jeugdhuis of jeugdcentrum",
+          "domain": "eventtype"
+        }
+      ],
+      ...
+    },
+    ...
+  ]
+}
+```
+
+## Embedding calendar summaries
+Formatting dates can be complex. Therefore we provide a parameter to expand the results from the API with a calendar summary for each result with calendar information (events, places).
+
+For every event or place a calendar summary is available in different sizes:
+-   `xs`: extra small
+-   `sm`: small
+-   `md`: medium
+-   `lg`: large
+
+We currently support 4 languages for the calendar summary: 
+-   `nl`
+-   `fr`
+-   `en`
+-   `de`
+
+The calendar summary can be retrieved in `text` and in `html`.
+
+<!-- theme: info -->
+> The response is always multilingual. It is possible to request multiple sizes and formats at once.
+>
+> If applicable, the calendar summary will also include the `status` or `booking availability` of the event or place.
+
+### Base URL
+
+```
+GET /{events|places|offers}
+```
+
+### Recommended use
+-   We recommend to use the format `md` for the search results but `lg` for events with calendarType `single` specifically
+-   We recommend to use `lg` for the detailpage
+-   In some cases (e.g. mobile apps) calendar summary `xs` can be useful
+-   You can use the `embedCalendarSummaries` parameter with or without `embed=true`
+
+### Parameters
+| Parameter                     | Required? | Type   | Possible values                         | Example                            |
+|:-------------------------------|:-----------|:--------|:-----------------------------------------|:------------------------------------|
+| embedCalendarSummaries (text) | optional  | string | `xs-text` `sm-text` `md-text` `lg-text` | `embedCalendarSummaries[]=sm-text` |
+| embedCalendarSummaries (html) | optional  | string | `xs-html` `sm-html` `md-html` `lg-html` | `embedCalendarSummaries[]=lg-html` |
+
+
+
+### Examples
```

Comment:
A lot less examples are included compared to the current documentation (https://documentatie.uitdatabank.be/content/integreren-met-searchv3/latest/calendarsummary.html), but the examples themselves are more relevant in my opinion:
* one example for `text`
* one example for `text` + `status`
* one example for `html` + `bookingAvailability`
* one example for requesting `multiple calendar summaries at once`